oak-store-composite/src/main/java/org/apache/jackrabbit/oak/composite/checks/ErrorHolder.java

```diff
@@ -27,16 +27,16 @@ class ErrorHolder {
     private static final int FAIL_IMMEDIATELY_THRESHOLD = 100;
     private final List<String> errors = new ArrayList<>();
     
-    public void report(MountedNodeStore mountedStore, String path, String error) {
-        errors.add(String.format("For NodeStore mount %s, path %s, encountered the following problem: '%s'", mountedStore.getMount().getName(), path, error));
+    public void report(MountedNodeStore mountedStore, String path, String error, MountedNodeStoreChecker<?> checker) {
```

Comment:
I think 'not versioned but exported' is the general Oak policy. I don't agree, but this is what we settled on.

For this particular change I have no backwards compatibility concerns since I did not see any exported APIs changes.

Reply:
> I did not see any exported APIs changes.

Well, ErrorHolder is exported, and the signature of one of its public methods changed with this PR!

Reply:
Sorry for the noise, ErrorHolder is package -private